docs/real-world-examples/ccfraud.md

```diff
@@ -132,3 +132,17 @@ This sample can be ran in distributed fashion, using [PyTorch Data Distributed P
 - If your cluster can be scaled to more than 1 machine, you can modify `config.yaml` in `examples/pipelines/ccfraud/` by adding `instance_count` to your silo config with number of nodes you would like to run the training on
 
 After performing all these steps you can rerun the experiment and it will run using DDP.
+
+## Communication encryption
+
+The communication between nodes, which includes intermediate outputs and gradients, can be optionally encoded. This may be very important in case of using *Redis* as communication backend. To enable the encryption follow these steps:
+
+1. Open the config file  `config.yaml` in `examples/pipelines/ccfraud/`
```

Comment:
is it `examples/pipelines/ccfraud_vertical/`?

Reply:
moved to `vertical-fl.md` doc